code/drasil-docLang/lib/Drasil/DocumentLanguage.hs

```diff
@@ -179,7 +178,7 @@ fillReferences dd si@SI{_sys = sys} = si2
     -- set new reference table in the chunk database
     chkdb2 = set refTable (idMap $ nub $ refsFromSRS ++ inRefs
       ++ map (ref.makeTabRef'.getTraceConfigUID) (traceMatStandard si) ++ secRefs -- secRefs can be removed once #946 is complete
-      ++ traceyGraphGetRefs (filter (not.isSpace) $ abrv sys) ++ map ref cites
+      ++ traceyGraphGetRefs (programName sys) ++ map ref cites
```

Comment:
The first of many similar changes. `filter (not.isSpace)` is often applied to the names of our various 'programs' (case study abbreviations) to remove spaces from the name. The goal of the filter was to make the program names friendly to code and file-systems. So, instead, I made it default to the calculated program names, which now remove spaces instead of replacing them with underscores.